.github/actions/pull_request_semver_label_checker.yml

```diff
@@ -0,0 +1,9 @@
+name: 'Pull request semver label checker'
+description: 'Checks that at least one semver label is applied to the pull request'
+runs:
+    using: "composite"
+    steps:
+      - name: Check labels
+        run: |
+            gh pr view 2809 --repo apple/swift-nio --json labels \
+            | jq -e '[.labels[].name] | any(. == "semver/major" or . == "semver/minor" or . == "semver/patch" or . == "semver/none")'
```

Comment:
Also do we want this to be in a script?

Reply:
I intentionally left it inline here because it is such a small thing and scripting didn't provide any meaningful local impact